packages/expo-build-properties/src/withBuildProperties.ts

```diff
@@ -0,0 +1,24 @@
+import type { ConfigPlugin } from '@expo/config-plugins';
+
+import { withAndroidBuildProperties, withAndroidProguardRules } from './android';
+import { withIosBuildProperties, withIosDeploymentTarget } from './ios';
+import { PluginConfigType, validateConfig } from './pluginConfig';
+
+/**
+ * Config plugin to customize native Android or iOS build properties for managed apps
+ * @param config ExpoConfig
+ * @param props `PluginConfig` from app.json or app.config.js
```

Comment:
```suggestion
 * @param props `PluginConfigType` from app.json or app.config.js
```
unclear how `PluginConfigType` is _from_ the app.json.